File: src/main/Python/boj_step2/boj14681.py
import sys

x = int(sys.stdin.readline())
y = int(sys.stdin.readline())
result = { (x > 0 and y > 0): 1, (x < 0 and y > 0): 2, (x < 0 and y < 0): 3, (x > 0 and y < 0): 4 }
print(result.get(True))

File: src/main/Cpp/boj_step2/boj14681.cpp
#include <iostream>
using namespace std;

int main() {
    int x, y;
    cin >> x >> y;
    int result;
    if (y > 0) {
        if (x > 0) result = 1;
        else result = 2;
    } else {
        if (x < 0) result = 3;
        else result = 4;
    }
    cout << result;
}